iocore/hostdb/HostDB.cc

```diff
@@ -1479,7 +1479,7 @@ HostDBContinuation::backgroundEvent(int /* event ATS_UNUSED */, Event * /* e ATS
     return EVENT_CONT;
   }
 
-  if ((hostdb_current_timestamp - hostdb_last_timestamp) > hostdb_hostfile_check_interval) {
+  if ((hostdb_current_timestamp.load() - hostdb_last_timestamp) > hostdb_hostfile_check_interval) {
```

Comment:
I don't think we need to call `.load()` here.

Reply:
I thought it too, but clang (15.0.1) complains without the `.load()`.
```
HostDB.cc:1482:33: error: invalid operands to binary expression ('std::atomic<ts_time>' (aka 'atomic<time_point<std::chrono::system_clock>>') and 'ts_time' (aka 'time_point<std::chrono::system_clock>'))
  if ((hostdb_current_timestamp - hostdb_last_timestamp) > hostdb_hostfile_check_interval) {
       ~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~
/usr/local/opt/llvm/include/c++/v1/__iterator/reverse_iterator.h:296:1: note: candidate template ignored: could not match 'reverse_iterator' against 'atomic'
operator-(const reverse_iterator<_Iter1>& __x, const reverse_iterator<_Iter2>& __y)
^
/usr/local/opt/llvm/include/c++/v1/__chrono/duration.h:435:1: note: candidate template ignored: could not match 'duration' against 'atomic'
operator-(const duration<_Rep1, _Period1>& __lhs, const duration<_Rep2, _Period2>& __rhs)
^
/usr/local/opt/llvm/include/c++/v1/__chrono/time_point.h:227:1: note: candidate template ignored: could not match 'time_point' against 'atomic'
operator-(const time_point<_Clock, _Duration1>& __lhs, const duration<_Rep2, _Period2>& __rhs)
^
/usr/local/opt/llvm/include/c++/v1/__chrono/time_point.h:238:1: note: candidate template ignored: could not match 'time_point' against 'atomic'
operator-(const time_point<_Clock, _Duration1>& __lhs, const time_point<_Clock, _Duration2>& __rhs)
^
/usr/local/opt/llvm/include/c++/v1/__iterator/move_iterator.h:283:6: note: candidate template ignored: could not match 'move_iterator' against 'atomic'
auto operator-(const move_iterator<_Iter1>& __x, const move_iterator<_Iter2>& __y)
     ^
/usr/local/opt/llvm/include/c++/v1/__iterator/wrap_iter.h:245:6: note: candidate template ignored: could not match '__wrap_iter' against 'atomic'
auto operator-(const __wrap_iter<_Iter1>& __x, const __wrap_iter<_Iter2>& __y) _NOEXCEPT
     ^
/usr/local/opt/llvm/include/c++/v1/__ios/fpos.h:61:11: note: candidate template ignored: could not match 'fpos' against 'atomic'
streamoff operator-(const fpos<_StateT>& __x, const fpos<_StateT>& __y) {
          ^
1 error generated.
```

Reply:
Gotcha.